bot/plugins/instance.py

```diff
@@ -395,41 +469,69 @@ async def execute(self) -> None:
             await plugin.app.rest.trigger_typing(self.channel)
 
         # try to execute code
-        att: hikari.Bytes | None = None
-        out: str | None
+        code_attr: hikari.Bytes | None = None
+        stdin_attr: hikari.Bytes | None = None
+        out: list[str] = [f"<@{self.requester}>"]
+
+        code_output_in_file = False
+        stdin_in_file = False
```

Comment:
Isn't this redundant, since code_file and stdin_file are None for false and Bytes for true?

Reply:
Doing it like this means I only need to assign the bytes class to a variable in one place.

Reply:
Oh I see